mmv1/templates/terraform/examples/kms_ekm_connection_basic.tf.erb

```diff
@@ -0,0 +1,33 @@
+resource "google_kms_ekm_connection" "<%= ctx[:primary_resource_id] %>" {
+  name            	= "<%= ctx[:vars]['ekmconnection_name'] %>"
+  location		      = "us-central1"
+  keyManagementMode = MANUAL
+  serviceResolvers  = [
```

Comment:
```suggestion
  service_resolvers  = [
```

Can all of the test blocks have field names made snake case? Thanks

Reply:
Sure, done.